src/QMCWaveFunctions/lcao/LCAOrbitalBuilder.cpp

```diff
@@ -435,9 +435,12 @@ namespace qmcplusplus
 
     std::vector<bool> corrCenter(num_centers, "true");
 
+    //What's this grid's lifespan?  Why on the heap?
```

Comment:
I think it can go on the stack instead.   OneDimQunticSpline only uses it to set a type (??), and MultiQuinticSpline grabs some values from the grid and doesn't keep a copy of the pointer (it should take a const reference instead).